tests/core/base/_utils/test_get_image_type.py

```diff
@@ -0,0 +1,30 @@
+"""Tests the `_utils.get_image_type` module."""
+
+import pytest
+
+from mirascope.core.base._utils.get_image_type import get_image_type
+
+
+def test_get_image_type() -> None:
+    """Test the get_image_type function."""
+    assert get_image_type(JPEG_DATA) == "jpeg"
+    assert get_image_type(PNG_DATA) == "png"
+    assert get_image_type(GIF_DATA) == "gif"
+    assert get_image_type(WEBP_DATA) == "webp"
+    assert get_image_type(HEIC_DATA) == "heic"
+    assert get_image_type(HEIF_DATA) == "heif"
+    with pytest.raises(ValueError, match="Unsupported image type"):
+        get_image_type(b"invalid")
+
+
+JPEG_DATA = b"\xff\xd8\xff\xe0\x00\x10JFIF\x00\x01\x02\x01\x00e\x00e\x00\x00\xff\xe1\x10>Exif\x00\x00"
```

Comment:
same here, do we want constants to be at bottom of file?

Reply:
Nope, see other response, this was originally so we could see the test function when the file data was way too long